sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/pipeline/pipeline_job.py

```diff
@@ -345,19 +345,30 @@ def _try_get_data_binding(_input_output_data) -> Union[str, None]:
                     return None
                 return _input_output_data._data_binding()
 
+            def _is_group_input(_input_obj) -> bool:
+                # exclude group input for now as it only comes from pipeline input now
+                # TODO: update validate init/finalize logic after supporting group as output
+                return not hasattr(_input_obj, "_data")
+
             _validate_job = self.jobs[_validate_job_name]
             # no input to validate job
             for _input_name in _validate_job.inputs:
-                _data_binding = _try_get_data_binding(_validate_job.inputs[_input_name]._data)
+                _input = _validate_job.inputs[_input_name]
+                if _is_group_input(_input):
+                    continue
```

Comment:
Skip or `_input.flatten()` to get the inner inputs?

Reply:
Add this in getting data binding function, use `flatten()` and pick first data binding for later use.